static/sass/features/launch.scss

```diff
@@ -12,7 +12,6 @@
     margin-bottom: 15px;
 
     > div {
-      @include gradient-bar-bg;
       background: rgba(255,255,255,0.8);
```

Comment:
should this to changed to one of the variables as well?

Reply:
Good catch.  Done.  

Screenshot: https://screenshot.googleplex.com/5mv5m457pUGhXhB